src/controller/org.controller/index.js

```diff
@@ -318,18 +313,88 @@ router.put('/org/:shortname',
   parseError,
   parsePostParams,
   controller.ORG_UPDATE_SINGLE)
+router.get('/org/:shortname/id_quota',
+  /*
+  #swagger.tags = ['Organization']
+  #swagger.operationId = 'orgIdQuota'
+  #swagger.summary = "Retrieves an organization's CVE ID quota information (accessible to all users)"
+  #swagger.description = "
+        <h2>Access Control</h2>
+        <p>All registered users can access this endpoint</p>
+        <h2>Expected Behavior</h2>
+        <p><b>Regular, CNA & Admin Users:</b> Retrieves CVE ID quota information for user's organization</p>""
```

Comment:
There's an extra quotation mark a the end so the Secretariat line isn't appearing and nothing in the Users section is appearing